stories/Histogram.stories.tsx

```diff
@@ -32,17 +34,95 @@ const gaussianMax = Math.max(...gaussianSample);
 // take about half of the values
 const randomFilter = () => Math.random() > 0.5;
 
-interface Style extends CSSProperties {
-  // TODO: define and extend the supported custom properties in franklin
-  // TODO: find a way to expose them globally when using franklin elements
-  '--main-histogram-color': string;
-  '--histogram-bar-gap': string;
-}
+type OtherArgs = { color: string; barGap?: string };
+
+type WithNBins = FC<
+  Omit<ComponentProps<typeof HistogramComponent>, 'position'> &
+    OtherArgs & {
+      nBins: number;
+      binSize: never;
+    }
+>;
+
+type WithBinSize = FC<
+  Omit<ComponentProps<typeof HistogramComponent>, 'position'> &
+    OtherArgs & {
+      nBins?: number;
+      binSize: number;
+    }
+>;
+
+const meta: Meta<WithNBins> = {
+  component: HistogramComponent,
+  title: 'Visualisation/Histogram',
+  parameters: {
+    purposeFunction: {
+      purpose: 'purpose',
```

Comment:
I noticed they were empty in few places. Shall we remove it or leave it empty?

Reply:
Removed everywhere